File: project/src/main/java/com/expedia/ui/javademo/interfaces/VHClickable.java
package com.expedia.ui.javademo.interfaces;

import android.view.View;

import com.expedia.ui.javademo.ItemVH;

/**
 * Created by nbirla on 15/11/17.
 */

public interface VHClickable {
    void onViewHolderClicked(ItemVH holder, View view);
}

File: project/src/main/java/com/expedia/ui/recyclerview/interfaces/VHClickable.kt
package com.expedia.ui.recyclerview.interfaces

import android.view.View
import com.expedia.ui.recyclerview.ItemVH

/**
 * Created by nbirla on 15/11/17.
 */

interface VHClickable {
    fun onViewHolderClicked(holder: ItemVH<*>, view: View)
}